lib/smart_params.rb

```diff
@@ -6,7 +6,7 @@
 
 module SmartParams
   extend ActiveSupport::Concern
-  include Dry::Types.module
+  include Dry.Types()
```

Comment:
Is this a new way to include things?

Reply:
Yeah dry-types pops a deprecation message recommending a switch `include Dry.Types()`